dora/stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java

```diff
@@ -67,13 +67,17 @@ public class StressWorkerBench extends AbstractStressBench<WorkerBenchTaskResult
 
   private FileSystem[] mCachedFs;
   private Path[] mFilePaths;
-  private Integer[] mOffsets;
-  private Integer[] mLengths;
   private FileSystemContext mFsContext;
 
-  /** generate random number in range [min, max] (include both min and max).*/
-  private Integer randomNumInRange(Random rand, int min, int max) {
-    return rand.nextInt(max - min + 1) + min;
+  /**
+   * generate random number in range [min, max] (include both min and max).
+   */
+  private long randomNumInRange(long min, long max) {
+    return ThreadLocalRandom.current().nextLong(min, max + 1) + min;
```

Comment:
note we are using thread local random here, so does the `--seed` parameter still take effect?

Reply:
i think we no longer need this param